src/Backend/Modules/MediaGalleries/Engine/Model.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Backend\Modules\MediaGalleries\Engine;
+
+use Backend\Core\Language\Language;
+use Backend\Core\Engine\Model as BackendModel;
+
+/**
+ * In this file we store all generic functions that we will be using in the MediaGalleries module.
+ */
+class Model
+{
+    /**
+     * Checks the settings and optionally returns an array with warnings
+     *
+     * @return array
+     */
+    public static function checkSettings(): array
```

Comment:
a bit a strange place for this method, also you can have none or 1 warning, why return an array?